vocabularies/Common.xml

```diff
@@ -622,6 +622,9 @@ The minimum structure is
         <Property Name="CollectionRoot" Type="Edm.String" Nullable="true">
           <Annotation Term="Core.Description" String="Service root of the value list collection; not specified means local to the document containing the annotation" />
         </Property>
+        <Property Name="DistinctValuesSupported" Type="Edm.Boolean" Nullable="true" DefaultValue="false"> 
```

Comment:
```suggestion
        <Property Name="DistinctValuesSupported" Type="Edm.Boolean" DefaultValue="false"> 
```

Reply:
Nullable defaults to true in CSDL XML, so we have to explicitly specify it.

And we want to always explicitly specify it for properties in vocabularies so that in a few months we still know what we intended.